src/commands/install-modules.yml

```diff
@@ -0,0 +1,70 @@
+description: >
+  Install your node modules with automated caching and best practices applied.
+
+parameters:
+  pkg-manager:
+    type: enum
+    enum: ["npm", "yarn"]
+    default: "npm"
+    description: Select the default node package manager to use.
+  app-dir:
+    type: string
+    default: "~/project"
+    description: Path to the directory containing your package.json file. Not needed if package.json lives in the root.
+  override-ci:
+    type: boolean
+    default: false
+    description: |
+      By default, modules will be installed with "npm ci" or "yarn install --frozen-lockfile", this override will use the standard "install" command.
+  with-cache:
+    type: boolean
+    default: true
+    description: Enable automatic caching of your node dependancies for increased speed.
+  cache-version:
+    type: string
+    default: v1
+    description: Change the default cache version if you need to clear the cache for any reason.
+  cache-key:
+    type: string
+    default: package.json
+    description: If this file is updated a new cache bucket will be created. Recommended to use package.json.
```

Comment:
> cache bucket

I wasn't aware of this term, is this what we're using? If so, should we make sure we use this terminology throughout our other orbs?

Reply:
Open to alternative descriptions?

Reply:
No, I'm not opposed to using this at all. Just wanted to know if this is our official term so I can use it in other places for consistency.